src/components/tasks/card/card.module.scss

```diff
@@ -97,7 +107,77 @@ $black: #252525;
 .progressYellow {
   background-color: $yellow;
 }
+
 .editButton {
   text-align: left;
   cursor: pointer;
 }
+
+.dropdownMenu {
+  position: relative;
+}
+
+.assigneeInputWrapper{
+   position: relative;
+    .currentAssignee{
+      position: absolute;
+      display: flex;
+      flex-wrap: nowrap;
+      justify-content: space-between;
+      top: -95%;
+      width:98%;
+      background-color: #041187;
+      color: $white;
+      border-radius: .25em;
+      padding: .5em;
+      padding-bottom: 1em;
+      .currentAssigneeName{
+        font-size: .85rem;
+        margin: 0;
+      }
+      button{
+        color: red;
+        border-radius:50%;
+        font-size: 1rem;
+        border: none;
+        cursor: pointer;
+      }
+    }
+  .assigneeInput{
+    font-family: inherit;
+    padding: 0.25em;
+    border: 2px solid #8b8a8b;
+    border-radius: .25em;
```

Comment:
It's better to follow 0.x rather than .x unless specified in docs

Reply:
sorry i didn't get what 0.x is and which doc you're referring to.

Reply:
got it will make the change